recipes/omero-py/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "omero-py" %}
+{% set version = "5.10.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/omero-py-{{ version }}.tar.gz
+  sha256: 4fbf7288586093d45e8f3a661ddf46143f38231017fada39c4e1c26725b34e33
+
+build:
+  entry_points:
+    - omero=omero.main:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - appdirs
+    - future
+    - numpy
+    - pillow
+    - python
+    - pywin32  # [win]
+    - pyyaml
+    - requests
+    - zeroc-ice <3.7
```

```suggestion
    - zeroc-ice =3.6
```
`<3.6` won't work.

Could optionally make it `zeroc-ice >=3.6.5,<3.7` since older versions probably won't work anyway?